sycl/include/sycl/ext/intel/pipes.hpp

```diff
@@ -76,22 +75,22 @@ template <class _name, class _dataT, int32_t _min_capacity = 0> class pipe {
 
   // Writing to pipe is lowered to SPIR-V instruction OpWritePipe via SPIR-V
   // friendly LLVM IR.
-  static void write(const _dataT &_Data) {
+  static void write(const DataT &Data) {
 #ifdef __SYCL_DEVICE_ONLY__
-    __ocl_WPipeTy<_dataT> _WPipe =
-        __spirv_CreatePipeFromPipeStorage_write<_dataT>(&m_Storage);
-    __spirv_WritePipeBlockingINTEL(_WPipe, &_Data, m_Size, m_Alignment);
+    __ocl_WPipeTy<DataT> _WPipe =
+        __spirv_CreatePipeFromPipeStorage_write<DataT>(&m_Storage);
+    __spirv_WritePipeBlockingINTEL(_WPipe, &Data, m_Size, m_Alignment);
 #else
-    (void)_Data;
+    (void)Data;
     throw sycl::exception(
         sycl::make_error_code(sycl::errc::feature_not_supported),
         "Pipes are not supported on a host device.");
 #endif // __SYCL_DEVICE_ONLY__
   }
 
 private:
-  static constexpr int32_t m_Size = sizeof(_dataT);
-  static constexpr int32_t m_Alignment = alignof(_dataT);
+  static constexpr int32_t m_Size = sizeof(DataT);
+  static constexpr int32_t m_Alignment = alignof(DataT);
```

Comment:
Looks ok for types. Should we also align style for class members naming? (example https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/accessor.hpp#L240)

Reply:
Interesting. Is this common style in Clang/LLVM to have special names like `M` or `m_` prefix for private members?

Reply:
Hello, I am not aware about strict rule for private members but general rule is to use CamelCase for variables:
"In general, names should be in camel case (e.g. TextFileReader and isLValue())"
and underscores is used as exception, not as rule:
"As an exception, classes that mimic STL classes can have member names in STL’s style of lower-case words separated by underscores (e.g. begin(), push_back(), and empty()). Classes that provide multiple iterators should add a singular prefix to begin() and end() (e.g. global_begin() and use_begin())."

https://llvm.org/docs/CodingStandards.html#name-types-functions-variables-and-enumerators-properly

Reply:
To clarify - existing commit looks good to be but I suggest to consider aligning class members names in separate commit too.

Reply:
We could change this later. The fpga compiler team will need to investigate this more since there might be a reason that we use m_prefixes (probably historical reasons).